src/client/Microsoft.Identity.Client/Platforms/Features/WebView2WebUi/WinFormsPanelWithWebView2.cs

```diff
@@ -214,10 +214,12 @@ private bool CheckForEndUrl(Uri url)
             if (!readyToClose &&
                 !url.Scheme.Equals("https", StringComparison.OrdinalIgnoreCase) &&
```

Comment:
minor: can we have url extension where this validation is performed. So the same code in two files will be centralized and any further changes will be at one place.